lib/Parser/ptree.cpp

```diff
@@ -578,6 +578,7 @@ ParseNodeTry::ParseNodeTry(OpCode nop, charcount_t ichMin, charcount_t ichLim)
 ParseNodeCatch::ParseNodeCatch(OpCode nop, charcount_t ichMin, charcount_t ichLim)
     : ParseNodeStmt(nop, ichMin, ichLim)
 {
+    this->pnodeParam = nullptr;
```

Comment:
hmmm interesting. Could you initialize this at initialization list of the constructor?